docs/reference/sql/limitations.asciidoc

```diff
@@ -113,3 +113,28 @@ FROM (SELECT ...) WHERE [simple_condition]`, this is currently **un-supported**.
 Using `FIRST` and `LAST` in the `HAVING` clause is not supported. The same applies to
 <<sql-functions-aggs-min,`MIN`>> and <<sql-functions-aggs-max,`MAX`>> when their target column
 is of type <<keyword, `keyword`>> as they are internally translated to `FIRST` and `LAST`.
+
+[float]
+=== Using TIME data type in GROUP BY or <<sql-functions-grouping-histogram>>
+
+Using `TIME` data type as a grouping key is currently not supported. For example:
```

Comment:
This should be a `NOTE:` or `WARNING:`

Reply:
We don't use them in the limitations page. In the `grouping.asciidoc` there is and `IMPORTANT` paragraph mentioning the same limitation.